manifests/a/Avangelista/CowabungaLite/1.0/Avangelista.CowabungaLite.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Avangelista.CowabungaLite
+PackageVersion: "1.0"
+InstallerLocale: en-US
+InstallerType: zip
+ReleaseDate: 2023-09-25
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: './CowabungaLite/CowabungaLite.exe'
+    PortableCommandAlias: cowabungalite
+  InstallerUrl: https://github.com/Avangelista/CowabungaLiteWindows/releases/download/1.0/CowabungaLite.zip
+  InstallerSha256: 7EB2BAB003604682E9D5A9748A29AEC5FAB138AC4E2267E159C988A34ADE9E24
+  Dependencies:
+    PackageDependencies:
+      - PackageIdentifier: 9PB2MZ1ZMB1S # Apple.iTunes It needs iTunes not directly to install but to work when using it

```

Comment:
* Related to https://github.com/microsoft/winget-cli/issues/3271

Cross-source dependencies won't always work here because not all users will have the msstore source (some may have removed it)

cc @denelon

Reply:
Dependencies in a manifest are only supported from the same source.